recipes/sentry-native/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SentryNativeConan(ConanFile):
+    name = "sentry-native"
+    description = "The Sentry Native SDK is an error and crash reporting client for native applications,\n" \
+                  "optimized for C and C++. Sentry allows to add tags,\n" \
+                  "breadcrumbs and arbitrary custom context to enrich error reports."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getsentry/sentry-native"
+    license = "MIT"
+    topics = ("conan", "breakpad", "crashpad",
+              "error-reporting", "crash-reporting")
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "backend": ["none", "inproc", "crashpad", "breakpad"],
+        "transport": ["none", "curl", "winhttp"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "backend": "inproc",
+        "transport": "curl"
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    _cmake = None
+
+    def requirements(self):
+        if self.options.transport == "curl":
+            self.requires("libcurl/7.68.0")
+            
+        if self.options.backend == "crashpad":
+            raise ConanInvalidConfiguration("crashpad not available yet in CCI")
+        if self.options.backend == "breakpad":
+            raise ConanInvalidConfiguration("breakpad not available yet in CCI")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        if self.options.backend == "inproc" and self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("The in-process backend is not supported on Windows")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["SENTRY_BACKEND"] = self.options.backend
+        self._cmake.definitions["SENTRY_ENABLE_INSTALL"] = True
+        self._cmake.definitions["SENTRY_TRANSPORT"] = self.options.transport
```

Comment:
```suggestion
        self._cmake.definitions["SENTRY_TRANSPORT"] = self.options.transport
        self._cmake.definitions["SENTRY_PIC"] = self.options.get_safe("fPIC", False)
```
because it forces fPIC option